cg_hermes/config/taxprofiler.py

```diff
@@ -1,90 +1,97 @@
 """Tags that are defined in Taxprofiler deliverables mapped to tags stored in Housekeeper."""
 
 from cg_hermes.config.nextflow import NEXTFLOW_TAGS
+from cg_hermes.constants.tags import (
+    TaxprofilerTags,
+    AnalysisTags,
+    ReportTags,
+    QCTags,
+    BioinfoToolsTags,
+)
 
 TAXPROFILER_COMMON_TAGS = {
     frozenset({"kraken2_combined_report", "kraken2"}): {
         "is_mandatory": True,
         "bundle_id": True,
-        "tags": ["kraken2", "combined-report"],
+        "tags": [TaxprofilerTags.KRAKEN2, TaxprofilerTags.COMBINED_REPORT],
         "used_by": ["clinical-delivery", "long-term-storage"],
```

Comment:
We also have constants for the used by fields 🙏

Reply:
Updated the PR!

Reply:
Updated the description after testing, I will merge.